index.html

```diff
@@ -0,0 +1,152 @@
+            <li class="nav_items"><a href="about.html">About</a></li>
+            <li class="nav_items"><a href="">Program</a></li>
+            <li class="nav_items"><a href="">Join</a></li>
+            <li class="nav_items"><a href="">Sponsor</a></li>
+          </ul>
+        </nav>
+      </header>
+      <div class="main-headline">
+        <h2 class="welcome">Welcome Humans</h2>
+        <h3 class="welcome-2">World is moving with</h3>
+        <h1 class="Meco">MECO</h1>
+        <p>
+          Bringing the 
+          change together 
+          with A.I. What you show They do learn 
+          The tech world is constantly evolving,with technologies
+          emerging regularly. <br>You are a Part of them <br>tooo!
+        </p>
+        <h3 class="bottom">
+          We are glad to share you that A joyful celebration believing in the
+          value of Machine Learning and A.I., creating a positive change with
+          people from over 80 countries is taking place in October, in Korea.
+        </h3>
+        <h2></h2>
+      </div>
+    </section>
+    <section class="program">
+        <h2 class="section-heading">Main Program</h2>
+            <div class="red-bar"></div>
+            <div class=" main-prog-items">
+                <div class="prog-items">
+                    <img src="./Images/program1.png" alt="icon">
+                    <h2>Lecture&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</h2>
+                    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Eligendi, velit?</p>     
+                </div>
+                <div class="prog-items">
+                    <img src="./Images/program2.png" alt="icon">
+                    <h2>CC Exhibition</h2>
+                    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Eligendi, velit?</p>     
+                </div>
+                <div class="prog-items">
+                    <img src="./Images/program3.png" alt="icon">
+                    <h2>Forum&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</h2>
+                    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Eligendi, velit?</p>     
+                </div>
+                <div class="prog-items">
+                    <img src="./Images/program4.png" alt="icon">
+                    <h2>Workshop</h2>
+                    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Eligendi, velit?</p>     
+                </div>
+                <div class="prog-items">
+                    <img src="./Images/program5.png" alt="icon">
+                    <h2>CC Lingration</h2>
+                    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Eligendi, velit?</p>     
+                </div>
+            </div>
+            <button type="submit" class="register-btn">Register Now</button>
+            <a href="#" id="whole-program">See The Whole Program</a>
+    </section>
+    <section class="speaker_section" id="speaker_section">
```

Comment:
- [ ] kindly reduce the gap between the speakers and the partners section
![Screenshot (980)_LI](https://user-images.githubusercontent.com/65298048/233436300-de3460f2-29f5-40c3-894a-c1413203196e.jpg)